libraries/ui-shared/Form/TextField/index.jsx

```diff
@@ -26,6 +26,8 @@ class TextField extends Component {
     label: PropTypes.node,
     /** Element to place left of text input (leading icon) */
     leftElement: PropTypes.node,
+    /** The max. length of the text input */
```

Comment:
it's forwared to the `Input` component here https://github.com/shopgate/pwa/blob/bda150fde720c7f03437318d371bb904632ba439/libraries/ui-shared/Form/TextField/index.jsx#L109 . its child components have a `maxlength` prop. see here https://github.com/shopgate/pwa/blob/master/libraries/common/components/Input/components/SimpleInput.jsx#L226

it's on my list for refactoring, since the prop contains a typo. should be `maxLength`